### ISSUE TITLE
Updates to tooltip def

### ISSUE THREAD

**Issue by MelSumner** [w3c/accname#151]
In moving the tooltip def to accname, it seems like some issues were surfaced. 
In order to complete the move, I'm filing the questions/suggestions as a separate issue to be worked on. 

[Comment with questions](https://github.com/w3c/accname/pull/135/files#r784163093)

- [ ] What about SVG title element? (not attribute) Should the definition be broadened to include that? e.g. "Any host language feature that would result in a user agent generating a tooltip..."
- [ ] Would also need to change dfn from "Tooltip attribute" to "Tooltip".

**Comment by scottaohara** [comment]
hmm. i don't know if "tooltip" alone should be used.  I mean it makes sense, to drop the 'attribute' but is there another word that could be used to augment this?  seems weird to have a `tooltip` role and then a "tooltip" term that isn't the same, and then the general lexicon definition of "tooltip" which appears to mean "anything that just popups up with info or whatever the heck else in it".

vocabulary is hard :)

**Comment by accdc** [comment]
I like the proposal by  @MelSumner "Any host language feature that would result in a user agent generating a tooltip"

Perhaps we could work in a reference to "native tooltip" to differentiate this from simulated ones that use ARIA?

**Comment by scottaohara** [comment]
Agreed.  The use of feature is good. I like the suggestion of “native” for the term
